src/features/projectsV2/ProjectDetails/styles/ProjectInfo.module.scss

```diff
@@ -140,6 +141,9 @@
   gap: 6px;
   align-items: center;
   min-width: 50%;
+  a {
+    font-size: $fontXXSmall;
+  }
```

Comment:
Please add a class selector for the styles applied for the year, it keeps things clearer.